yb-voyager/cmd/importData.go

```diff
@@ -577,7 +577,54 @@ func importData(importFileTasks []*ImportFileTask) {
 	if importerRole == TARGET_DB_IMPORTER_ROLE {
 		importDataCompletedEvent := createSnapshotImportCompletedEvent()
 		controlPlane.SnapshotImportCompleted(&importDataCompletedEvent)
+		packAndSendImportDataPayload(COMPLETE) // TODO: later for other import data commands
+	}
+
+}
+
+func packAndSendImportDataPayload(status string) {
+	//TODO send this for INPROGRESS status in some fixed interval for long running import data
+	if !callhome.SendDiagnostics {
+		return
+	}
+	payload := createCallhomePayload()
+	switch importType {
+	case SNAPSHOT_ONLY:
+		payload.MigrationType = OFFLINE
+	case SNAPSHOT_AND_CHANGES:
+		payload.MigrationType = LIVE_MIGRATION
+	}
+	bytes, err := json.Marshal(targetDBDetails)
+	if err != nil {
+		log.Errorf("error in parsing sourcedb details: %v", err)
+	}
+	payload.TargetDBDetails = string(bytes)
+	payload.MigrationPhase = IMPORT_DATA_PHASE
+	importDataPayload := callhome.ImportDataPhasePayload{
+		ParallelJobs: int64(tconf.Parallelism),
+		StartClean:   bool(startClean),
 	}
+	importRowsMap, err := getImportedSnapshotRowsMap("target")
+	if err != nil {
+		log.Errorf("error in getting the import data: %v", err)
+	} else {
+		importRowsMap.IterKV(func(key sqlname.NameTuple, value int64) (bool, error) {
+			importDataPayload.TotalRows += value
+			if value > importDataPayload.LargestTableRows {
+				importDataPayload.LargestTableRows = value
+			}
+			return true, nil
+		})
+	}
+	importDataPayloadBytes, err := json.Marshal(importDataPayload)
+	if err != nil {
+		log.Errorf("error in parsing the export data payload: %v", err)
```

Comment:
Makes sense that we **ONLY** log the errors and do not return/fail. However, I think it would make sense to prefix the errors with "callhome:" so that we don't confuse them with actual errors. I don't suppose there is another way to determine that the errors are from call home, right now, is there? 

if you're making the change, pls make it for all the relevant functions and logs

Reply:
Yes sure that's a good idea to distinguish the logs. Thanks! will make the changes